src/pages/home/report/ReportActionItemMessageEdit.js

```diff
@@ -147,7 +153,7 @@ class ReportActionItemMessageEdit extends React.Component {
         // This component is rendered only when draft is set to a non-empty string. In order to prevent component
         // unmount when user deletes content of textarea, we set previous message instead of empty string.
         if (newDraft.trim().length > 0) {
-            this.debouncedSaveDraft(newDraft);
+            this.debouncedSaveDraft(_.escape(newDraft));
```

Comment:
I would also like to let you know that comment here will be helpful.

Reply:
Updated